src/NServiceBus.Testing/Saga.cs

```diff
@@ -6,6 +6,11 @@
     using System.Threading.Tasks;
     using MessageInterfaces.MessageMapper.Reflection;
 
+    internal static class SagaConsts
```

Comment:
Why not just a internal const field on the saga class? 

If we keep it you can remove the `internal` since that's the default